### PR TITLE
completed object-manipulation-2

### PR DESCRIPTION
**Pull request by amyaitran**
image

### CODE REVIEW

File: object-manipulation-2/src/invert.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// assign the source value as the output trait, and the source trait as the output value.
+// return the output.
+
+function invert(source) {
+  var output = {};
+  var sourceKeys = Object.keys(source);
+  var sourceValues = Object.values(source);
+  for (var i = 0; i < sourceKeys.length; i++) {
+    output[sourceValues[i]] = sourceKeys[i];
+  }
```
Comment (codingcodymiller) [inline object-manipulation-2/src/invert.js:16]: I think you can do this without needing to create arrays. This is a good opportunity for a `for...in` loop

File: object-manipulation-2/src/omit.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* exported omit */
+
+// create a storage for the output that holds a copy of the source object.
+// look at all the keys.
+// if the key exists as a trait in the source copy,
+// then delete that key from the source copy.
+// return the output.
+
+function omit(source, keys) {
+  var output = { ...source };
```
Comment (codingcodymiller) [inline object-manipulation-2/src/omit.js:10]: If you can explain what this is doing to me, I'll let you keep it 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! A couple notes for you:

File: object-manipulation-2/src/omit.js (hunk 2 above)
Comment (amyaitran) [reply object-manipulation-2/src/omit.js:10]: i'll come find you during one of these breaks 😅

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**